website/docs/about/architecture/architecture-archive-container.mdx

```diff
@@ -10,7 +10,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 The archive data container contains all the components that make up the ORCA
 ingest service functionality. The diagram below shows how the various components
 interact. A majority of the work for this functionality is done by the
-**copy_to_glacier** lambda function. This component was designed such that it
+**copy_to_archive** lambda function. This component was designed such that it
```

Comment:
Wondering if this should really be copy_to_orca as this is what the end user will actually see.

Reply:
This string matches the internal name rather than the public name. As such, added qualifier.